Formula/vorbis-tools.rb

```diff
@@ -25,16 +25,27 @@ class VorbisTools < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "b137af1a327bf93e3922244bf03ab2eef1151838483b2e71e1e35c5254d32cc4"
   end
 
+  # macOS `ld64` does not like the `.la` files created during the build.
+  depends_on "llvm" => :build if DevelopmentTools.clang_build_version >= 1400
```

Comment:
We should report this upstream. This most likely means some non-object files are being included in `.a` archives by mistake.